docs/ldml/tr35-general.md

```diff
@@ -960,25 +976,33 @@ The formal syntax for identifiers is provided below.
     <td>"kibi", "mebi", …
         <ul><li><em>Note: </em>See full list at <a href="https://physics.nist.gov/cuu/Units/binary.html">Prefixes for binary multiples</a></li></ul></td></tr>
 
-<tr><td>simple_unit</td><td>:=</td>
-    <td>unit_component ("-" unit_component)*<br/>
-        | “em” | “g” | “us” | “hg” | “of”
-        <ul><li><em>Example:</em> gallon-imperial</li>
-            <li><em>Constraint:</em> At least one unit_component must not itself be a simple_unit</li>
-            <li><em>Note:</em> 3 simple units are currently allowed as legacy usage, where a component wouldn’t be a unit_component (eg for “<strong>g</strong>-force”)
-                <ul><li>We will likely deprecate those and add conformant aliases in the future.</li>
-                    <li>“hg” and “of” are already only in deprecated simple_units.</li>
-                </ul></li>
-        </ul></td></tr>
-
-<tr><td>unit_component</td><td>:=</td>
+<tr><td>prefix_component</td><td>:=</td>
+    <td>[a-z]{3,∞}
+        <ul><li><em>Constraint:</em> must be value in: &lt;unitComponent type=”prefix_component”&gt;.</li></ul></td></tr>
+
+<tr><td>base_component</td><td>:=</td>
+    <td>[a-z]{3,∞}
+        <ul><li><em>Constraint:</em> must not be a value in: &lt;unitComponent type=”suffix_component”&gt; 
+			or &lt;unitComponent type=”suffix_component”&gt; 
```

Comment:
Question: what is a unitComponent? I don't see that term defined anywhere.

I would have thought to instead write this as, "Constraint: does not match any of the following: ..."

Reply:
It is a new structure in the XML file. Which is a good point; it needs to be documented. Will have subsequent PR for that.

Reply:
> I would have thought to instead write this as, "Constraint: does not match any of the following: ..."

That's better wording.